elastic/README.md

```diff
@@ -28,11 +28,15 @@ instances:
     pending_task_stats: true
 ```
 
-If you're collecting Elasticsearch metrics from just one Datadog Agent running outside the cluster — e.g. if you use a hosted Elasticsearch — set `cluster_stats` to true.
+**Note**:
+
+* If you're collecting Elasticsearch metrics from just one Datadog Agent running outside the cluster — e.g. if you use a hosted Elasticsearch — set `cluster_stats` to true.
+
+* To use the Agent's ElasticSearch integration for the Amazon Elasticsearch service, set the `url` parameter on a remote host to point to wherever your AWS elasticsearch stats url is.
```

Comment:
change to `to point to your AWS Elasicsearch stats URL.`

Reply:
Also the "on a remote host" bit is confusing and should probably be removed. So the entire sentence is:
` To use the Agent's Elasticsearch integration for the AWS Elasticsearch services, set the ``url`` parameter to point to your AWS Elasticsearch stats URL.`